SampleApps/CarthageTest/Cartfile

```diff
@@ -1 +1 @@
-github "braintree/braintree_ios"
\ No newline at end of file
+github "braintree/braintree_ios" "kount-module"
```

Comment:
random: does this make it a breaking change?

Reply:
This is the sample app and we have needed to point to the new branch for all of the new modules we make. Once `next-major-version` is merged into master/main we should be able to update this in that PR or in a fast follow. But merchants would not need to point to our branch or next major version once the entire feature branch is merged in, if that's the question.

Reply:
Got it nah I realize now that `kount-module` is a GitHub branch 🤦 . I thought this was a list of modules lol.

Reply:
Ah no worries, understandably confusing. Package managers :shakes_fist: